src/app/components/plugin-uiframe/plugin-uiframe.component.ts

```diff
@@ -273,6 +290,59 @@ export class PluginUiframeComponent implements OnChanges, OnDestroy {
         iframe?.contentWindow?.postMessage?.(message, this.pluginOrigin ?? "*");
     }
 
+    private loadImplementations(): void {
+        const firstPage = this.loadImplementationsFromPage(0);
+        
+        firstPage?.pipe(
+            map(firstPage => {
+                let pages: Observable<ApiObjectList<ExperimentDataApiObject>>[] = [of(firstPage)]
+                for (let i = 1; i < firstPage.itemCount / this.itemsPerPage; i++) {
+                    const page = this.loadImplementationsFromPage(i)
+                    if (page !== null) {
+                        pages.push(page)
+                    }
+                }
+                return pages;
+            }),
+            mergeAll(),
+            map(wholePage => 
+                wholePage.pipe(
+                    map(apiObjectList => apiObjectList.items.filter(experminetData => experminetData.contentType === "application/qasm" || experminetData.contentType === "application/qiskit")),
+                    map(dataItems => dataItems.map(item => this.experimentId ? this.backend.getExperimentData(this.experimentId, item.name, item.version) : undefined)),
+                    filter((experimentData): experimentData is Observable<ExperimentDataApiObject>[] => !!experimentData),
```

Comment:
use `boolean(experimentData)` for a more explicit conversion to boolean. (may also be Boolean or even new Boolean)